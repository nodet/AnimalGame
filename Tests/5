No
salmon
Does it live in water?
Quit
