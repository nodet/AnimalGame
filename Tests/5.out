Hi, let's play the animal game. You can quit at any time by typing "Quit."

Think of an animal.
Is your animal a cat? Oh no. What was it? What is a yes/no question to tell a salmon from a cat? Thanks! Let's play again.
Think of an animal.
Does it live in water? Ok. Bye!
cat
salmon
Does it live in water?
